R/utils-dat.R

```diff
@@ -62,10 +64,48 @@ dat_install <- function (os = 'win', path, ver = '13.10.0') {
   unzip(normalizedDest, exdir = normalizedPath)
   file.remove(normalizedDest)
 
+  message("Modifying system PATH to enable easier access to 'dat'...")
+
+  # These still use `system()` vs `system2()`
   if (os == 'win') {
     system(paste0('SETX /M PATH "%PATH%;',
      paste0(normalizedPath, destpath), '"'))
   } else {
     system(sprintf("echo 'export PATH=$PATH:%s' >> ~/.profile", normalizedPath))
   }
 }
+
+# tests whether we can find dat
+dat_is_installed <- function(quiet=TRUE) {
+
+  # try to get the dat version number since that's more robust than
+  # just executing it or trying to find the file since it means both
+  # the PATH is setup right and the utility is functioning.
+
+  # 'dat' does not conform to traditional cli standards and returns 1 on
```

Comment:
Anything upstream I could notify about this? I don't know what cli standards there are but happily relay info to the Dat team :)